doc/releases/release-notes-1.12.rst

```diff
@@ -752,7 +752,7 @@ These GitHub issues were closed since the previous 1.11.0 tagged release:
 * :github:`7979` - drivers: dma: dma_cavs: DMA driver does not support per-channel callbacks
 * :github:`7989` - eth: ping: can't ping others from zephyr console
 * :github:`8005` - FRDM-K64F boot hang w/ mcuboot + lwm2m client
-* :github:`8009` - POSIX `clock_gettime()` is discontinuous
+* :github:`8009` - POSIX clock_gettime() is discontinuous
```

Comment:
Some of these were converted from single ticks to: no ticks, double backticks, and single quotes

Not a big deal but it would be nice to pick one and be consistent